releases/latest/beta/Dockerfile.ubuntu.openjdk11

```diff
@@ -30,6 +30,8 @@ RUN apt-get update \
     && rm /tmp/wlp.zip.sha1 \
     && mkdir -p /licenses \
     && cp /opt/ol/wlp/LICENSE /licenses/ \
+    && wget -O /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_x86_64 \
```

Comment:
Would InstantOn be supported with Ubuntu-based Liberty images? If not, wondering whether this is needed

Reply:
Ubuntu based image support will come later.  This is just to make the UBI and ubuntu images equivalent so when the Semeru Ubuntu based images do support CRIU then our Ubuntu based images will work without changes.